connect-button/src/test/java/com/ifttt/connect/ui/DisableTrackingTest.java

```diff
@@ -0,0 +1,37 @@
+package com.ifttt.connect.ui;
+
+import android.app.Activity;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import com.ifttt.connect.ShadowAnimatorSet;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.annotation.Config;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@RunWith(AndroidJUnit4.class)
+@Config(shadows = { ShadowAnimatorSet.class })
+public class DisableTrackingTest {
+
+    private final Activity activity = Robolectric.buildActivity(TestActivity.class).create().get();
+    private AnalyticsManager analyticsManager;
+
+    @Before
+    public void setup() {
+        analyticsManager = AnalyticsManager.getInstance(activity);
```

Comment:
If you don't clear queue, can you still run the tests? Or is the queue not going to be cleared in between tests?

Reply:
Since this class has a single test, the test will work without the queue being explicitly cleared.